### PR TITLE
Ergänzung impact

### PR DESCRIPTION
**Pull request by mvglasow**
Vergleiche hierzu #23.

`impact` habe ich bislang in `roadworks` und `closures` beobachtet. Insofern ist es evtl. nicht 100% korrekt, das in `RoadEvent` unterzubringen – letzteres wird auch für `warning` verwendet, wo ich `impact` bislang noch nicht gesehen habe. Andererseits scheint `impact` auch dort, wo es verwendet wird, optional zu sein – nicht jedes Event verwendet es tatsächlich.

### CODE REVIEW

**Comment by LilithWittmann** [comment]
Hey @mvglasow thanks for your work on that. Could you maybe please have a look why schemathesis fails to validate your PR?
Thanks!

**Comment by mvglasow** [comment]
There are a few things amiss:

* `Content-Type` is not as expected, e.g. in https://github.com/bundesAPI/autobahn-api/actions/runs/4503233699/jobs/7926159574?pr=33#step:7:38 and (as far as I can tell) every single other check. I doubt it is related to the changes I made.
* https://github.com/bundesAPI/autobahn-api/actions/runs/4503233699/jobs/7926159574?pr=33#step:7:92 is interesting: `display_type` is `CLOSURE_TRUCKS`, which we were not aware of yet. Also not related to my changes, we just happened to stumble upon a previously unseen value just at the right moment. That’ll be a new issue and PR.

Bottom line, none of the two looks like my changes caused the failure.

Given that we are checking against a reverse-engineered API, which may change its behavior or reveal previously unknown features any time, it might be a good idea to run GHA on `main` periodically. If that fails all of a sudden, it’s a sign we missed something.

**Comment by mvglasow** [comment]
#36: `CLOSURE_TRUCKS`

#37: Periodic GHA to detect API changes
